### PR TITLE
Create README_es.MD

### PR DESCRIPTION
**Pull request by LuigiBaute**
I could also translate the contributing doc we can discuss that.

### CODE REVIEW

File: README_es.MD [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+
+`cfn-lint docs AWS::Lambda::Function` 
+
+`cfn-lint docs AWS::Lambda::Function.Code`
+
+`cfn-lint docs AWS::Lambda::Function.Code.S3Bucket`
+
+**Nota: El orden de `--parameters` y` --pseudo` actualmente importa. Esto debería abordarse en una nueva versión.**
+
+[//]: # (Translation Section: Example Output)
```
Comment (martysweet) [inline README_es.MD]: This won't be the translated output of the program without additional work (which would be outside the scope of this PR).

File: README_es.MD [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+
+[//]: # (Translation Section:What can cfn-lint do?)
+### ¿Qué puede hacer cfn-lint?
+* <tt>Read JSON + YAML (Including YAML short form)</tt> Lea JSON + YAML (Incluyendo YAML en su formato más pequeño)
+* <tt>Detect invalid property names</tt> Detecta nombres de propiedad inválido
+* <tt>Detect invalid Ref</tt> Detecta Ref inválido
+* <tt>Detect invalid mappings</tt> Detecta mapeos inválidas
+* <tt>Detect invalid format</tt> Detecta formato inválido
+* <tt>Missing or invalid AWSTemplateFormatVersion</tt> Faltante o inválido Versión de formato de plantilla AWS
+* <tt>Fn::FindInMap</tt> (Encontrar en mapa)
```
Comment (martysweet) [inline README_es.MD]: Can these be like the above, without the brackets ()?

ex.
`* <tt>Fn::FindInMap</tt> Encontrar en mapa`

**Comment by martysweet** [comment]
Can we add from Spanish -> English documentation and also a link from English to Spanish? 
I think something like the following would be best placed just before the Contributions section.

```
## Translations
This document is available in:

- [English](https://github.com/martysweet/cfn-lint/blob/master/README.md)
- [Español](https://github.com/martysweet/cfn-lint/blob/master/README_es.md)

If you wish to translate this document to another language, we welcome your contribution! 
```

File: README_es.MD (hunk 1 above)
Comment (LuigiBaute) [reply README_es.MD]: Should I delete this part?

**Comment by LuigiBaute** [comment]
I could also translate the contributing, the thing is that my contributions needs to have over a thousand words and the contributing has about +800 so I'd have to do the license too to achieve the goal of a thousand words. let me know if you have a better idea.

File: README_es.MD (hunk 1 above)
Comment (martysweet) [reply README_es.MD]: Nope, just restore the English version of the program output.

File: README_es.MD [hunk 3]
```diff
@@ -40,7 +40,6 @@ Si lo consigues `/usr/bin/env: 'node': No such file or directory (El archivo o d
 
 `cfn-lint docs AWS::Lambda::Function.Code.S3Bucket`
 
-**Nota: El orden de `--parameters` y` --pseudo` actualmente importa. Esto debería abordarse en una nueva versión.**
```
Comment (martysweet) [inline README_es.MD]: Why was this deleted?

File: README_es.MD [hunk 4]
```diff
@@ -281,7 +280,8 @@ Cada función intrínseca tiene una firma de `doIntrinsicXYZ (ref, key)` y es ll
 el valor resuelto de la función. Por ejemplo, para Fn :: Sub, una entrada de `"Mi $ {MiInstancia}"`
  devolvería una cadena similar a `"Mi i-0a0a0a0a0a`.
 
-[//]: # (Translation Section:Contributions)
-## Contribuciones
+## Translations
```
Comment (martysweet) [inline README_es.MD]: Can you leave the contributions section at the bottom of the document? Putting translations above it.

File: README_es.MD [hunk 5]
```diff
@@ -81,20 +80,20 @@ Documentación: http://docs.aws.amazon.com/search/doc-search.html?searchPath=doc
...
-* <tt>Fn::Select</tt> (Seleccionar)
-* <tt>Condition support</tt> (Condición de soporte)
+* <tt>Fn::If</tt> sí
+* <tt>Fn::Equals</tt> Iguales
+* <tt>Fn::Not</tt> No
+* <tt>Fn::Or</tt> O
+* <tt>Fn::And</tt> Y
+* <tt>Fn::ImportValue</tt> Importar valor
+* <tt>Fn::Select</tt> Seleccionar
+* <tt>Condition support</tt> Condición de soporte
```
Comment (martysweet) [inline README_es.MD]: Perfect :+1:

**Comment by LuigiBaute** [comment]
Done. I hope you like it, I'll adjust it as many times as necesary. :+1:

File: README_es.MD [hunk 6]
```diff
@@ -0,0 +1,293 @@
...
+
+`cfn-lint docs AWS::Lambda::Function.Code`
+
+`cfn-lint docs AWS::Lambda::Function.Code.S3Bucket`
+
+**Nota: El orden de `--parameters` y `--pseudo` actualmente importa. Esto debería abordarse en una nueva versión.**
+
+[//]: # (Translation Section: Example Output)
+### Ejemplo de salida
+```
```
Comment (martysweet) [inline README_es.MD]: Make this section

```
0 infos
0 warn
2 crit
Resource: Resources > MyInstance > Properties
Message: Required property ImageId missing for type AWS::EC2::Instance
Documentation: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-instance.html

Resource: Resources > MyInstance > Properties
Message: Required property Value missing for type Tag
Documentation: http://docs.aws.amazon.com/search/doc-search.html?searchPath=documentation-guide&searchQuery=Tag&x=0&y=0&this_doc_product=AWS+CloudFormation&this_doc_guide=User+Guide&doc_locale=en_us#facet_doc_product=AWS%20CloudFormation&facet_doc_guide=User%20Guide

Template invalid!
```

File: README_es.MD [hunk 7]
```diff
@@ -0,0 +1,293 @@
...
+el valor resuelto de la función. Por ejemplo, para Fn :: Sub, una entrada de `"Mi $ {MiInstancia}"`
+ devolvería una cadena similar a `"Mi i-0a0a0a0a0a`.
+
+[//]: # (Translation Section:Contributions)
+## Contribuciones
+
+Consulte la [Guía para colaboradores](CONTRIBUTING.md) para obtener más información sobre cómo comenzar.
+
+## Translations
+This document is available in:
```
Comment (martysweet) [inline README_es.MD]: Can there be Spanish for this and the header?

**Comment by LuigiBaute** [comment]
Hi @martysweet I made the changes. :+1:

**Comment by LuigiBaute** [comment]
HI! @martysweet :+1:

**Comment by martysweet** [comment]
Thanks for your contribution @LuigiBaute!

**Comment by LuigiBaute** [comment]
Thanks to you! I learn from every single project I make, but thanks to your reviews I learned more than usually! so thanks! Happy to deliver. :1st_place_medal:

**Comment by LuigiBaute** [comment]
@martysweet you are going to merge this right? just to know :+1:

**Comment by LuigiBaute** [comment]
hi @martysweet I'd love to keep contributing to this project but what I can offer from now is an adjust on the readme_es I've been arguing about some terms that I'd like to change. I'll be sending the new pull request by tomorrow probably.

**Comment by martysweet** [comment]
Hi @LuigiBaute, as this PR is closed, please use the Issue for further comments :) I'm happy to accept any changes!
